diff --git a/ByteBuffer.ORIGINAL.cpp b/ByteBuffer.cpp
index f37cbc1..f47c0c2 100644
--- a/ByteBuffer.ORIGINAL.cpp
+++ b/ByteBuffer.cpp
@@ -85,7 +85,7 @@ uint8_t ByteBuffer::putString(char *in){
 	char *inString;
 
 	inString=in;
-	cli();
+	//  cli();   // disabled by zach
 	while(length <= capacity){
 		if (length == capacity) {
 			fillError=true;
@@ -105,7 +105,7 @@ uint8_t ByteBuffer::putString(char *in){
 }
 
 uint8_t ByteBuffer::putInFront(byte in){
-	cli();
+	//  cli();   // disabled by zach
 	if(length < capacity){
 			// save data byte at end of buffer
 			if( position == 0 )
@@ -124,7 +124,7 @@ uint8_t ByteBuffer::putInFront(byte in){
 }
 
 byte ByteBuffer::get(){
-	cli();
+	//  cli();   // disabled by zach
 	byte b = 0;
 
 	if(length > 0){
@@ -140,7 +140,7 @@ byte ByteBuffer::get(){
 byte ByteBuffer::getFromBack(){
 	byte b = 0;
 	if(length > 0){
-		cli();
+	//  cli();   // disabled by zach
 		b = data[(position+length-1)%capacity];
 		length--;
 	}
@@ -263,4 +263,4 @@ float ByteBuffer::getFloatFromBack(){
 	return ret;
 }
 
-
+
